src/renderer/components/SideDrawer.js

```diff
@@ -211,15 +213,23 @@ export function SideDrawer({
                 p="24px"
                 style={{ zIndex: 200 }}
               >
-                {onRequestBack && (
+                {onRequestBack ? (
                   <Button onClick={onRequestBack} className="sidedrawer-close">
                     <IconAngleLeft size={12} />
                     <Text ff="Inter|Medium" fontSize={4} color="palette.text.shade40">
                       <Trans i18nKey="common.back" />
                     </Text>
                   </Button>
+                ) : (
+                  <Box />
```

Comment:
do you need an empty box here?

Reply:
Indeed, It's used in the same way as [this former Box](https://github.com/LedgerHQ/ledger-live-desktop/pull/3978/files#diff-4174bb1ccf94b6f908bbf947982f1b401f1f904e6e8401acb57accc155d4b663L222) `<Box flex="1" />` to maintain components position in this "header". 

But now, since we have 3 different components in this header (which can all be independently enable or not), we need to ensure we always have a component on each side of the header to maintain components position.
For example, we always want the cross to be on the right, even if we don't have back button or title (or only one of the two).